monitoring/status-page/status/routes.py

```diff
@@ -1,52 +1,90 @@
 import argparse
 import json
+import queue
 import requests
+from collections import namedtuple
 from flask import render_template
 from status import app
-from collections import namedtuple
+from threading import Thread
 
-NetworkInfo = namedtuple('NetworkInfo', ['name', 'watchdog', 'explorer', 'staking', 'endpoint'])
+NetworkInfo = namedtuple('NetworkInfo', ['watchdog', 'explorer', 'staking', 'endpoint'])
 
 watchdog = 'http://watchdog.hmny.io/status-%s'
 endpoint = 'https://api.s%s.%s.hmny.io'
 
+statuses = {}
+q = queue.Queue(maxsize = 0)
+
+
 @app.route('/status')
 def status():
 
     with open('networks.txt', 'r') as f:
```

Comment:
can you add a comment with a sample of the 'networks.txt' here so that we can maintain it?

Reply:
`networks.txt` is [here](https://github.com/harmony-one/harmony-ops/blob/master/monitoring/status-page/networks.txt). First line is the syntax.